packages/flutter/lib/src/services/binding.dart

```diff
@@ -68,8 +68,11 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
   void _initKeyboard() {
     _keyboard = HardwareKeyboard();
     _keyEventManager = KeyEventManager(_keyboard, RawKeyboard.instance);
-    platformDispatcher.onKeyData = _keyEventManager.handleKeyData;
-    SystemChannels.keyEvent.setMessageHandler(_keyEventManager.handleRawKeyMessage);
+    _keyboard.syncKeyboardState().then((_) {
+      debugPrint('_keyboard.syncKeyboardState().completed at ${DateTime.now}');
```

Comment:
debug print

Reply:
This was my last attempt to get more information about the web test failures because I cant' repro those failures locally 😅.
I reverted the commit where I added this debugPrint, I have no clue for the moment on how to tackle those test failures.